build-tools/build-release-images.sh

```diff
@@ -20,14 +20,12 @@ VERSION_INFO=$(${CURDIR}/version-tool version)
 
 # Hard code the platform dir here
 cp $CURDIR/../_docker_workspace/out/$RELEASE_PLATFORM/bin/* $WKDIR/
-mkdir -p $WKDIR/python
-cp python/*.py $WKDIR/python/
-cp python/k8s-runtime-requirements.txt $WKDIR/
+cp requirements.txt $WKDIR/
 cp schemas/bigip-virtual-server_v*.json $WKDIR/
 cp LICENSE $WKDIR/
 cp $CURDIR/help.md $WKDIR/help.md
 echo "{\"version\": \"${VERSION_INFO}\", \"build\": \"${BUILD_INFO}\"}" \
-  > $WKDIR/python/VERSION_BUILD.json 
+  > $WKDIR/VERSION_BUILD.json
```

Comment:
I think this is only used in the bigipconfigdriver, so I doubt you need to keep it here anymore.

Reply:
ya let me check how the driver actually uses this. Then I'll determine if it should be pulled out or not good catch though.

Reply:
This needs to stay but I am going to have to update the f5-ctlr-agent and then the git SHAs